active/PagerControl/PagerControl.md

```diff
@@ -0,0 +1,380 @@
+
+XAML
+```XAML
+<Grid>
+         <controls:DataGrid x:Name="dataGrid" 
+                Height="600" Margin="12"
+                AutoGenerateColumns="True"
+                ItemsSource="{x:Bind MyViewModel.Items}" /> 
+        <muxc:PagerControl x:Name="MainPagerControl"
+                DisplayMode="NumericalButtonPanel"
+	        NumberOfPages="10"
+                FirstButtonVisibility="None"
+                LastButtonVisibility="None"
+	        EllipsisShowFirstAndLast="True">
+	</muxc:PagerControl>
+</Grid>
+```
+
+### Customize edge buttons
+
+You can customize the visibility of each of the edge buttons (the first, previous, next, and last) by setting their visibility properties. In this example, the first and previous buttons will always be visible while the next button will be hidden when the last page is selected. The last button will be hidden. By default, all buttons will be set to AlwaysVisible.
+
+XAML
+```XAML
+<Grid>
+        <controls:DataGrid x:Name="dataGrid" 
+                Height="600" Margin="12"
+                AutoGenerateColumns="True"
+                ItemsSource="{x:Bind MyViewModel.Items}" /> 
+        <muxc:PagerControl x:Name="MainPagerControl"
+                DisplayMode="ComboBox"
+                NumberOfPages="10"
+                NextButtonVisibility="HiddenOnEdge"
+                LastButtonVisibility="None"/>
+        </muxc:PagerControl>
+</Grid>
+```
+
+### Code Example
+
+Below is a code sample for how to use the pager control with a data grid and how to handle the page changed event. Since this control is only a UI component, the data managment still needs to be handled through code. This example also shows how to set the number of pages property and manage what images need to be shown to the user when they select a specific page number. 
+
+When a user interacts with the pager control and selects an edge button or page number, the page changed event will fire. This event will provide the index the user is coming from (PreviousPageIndex) and the index the user is navigating to (NewPageIndex). 
+
+XAML
+```XAML
+<Page
+    x:Class="WindowsGallaryApp.Samples.DataGridSamplePage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:WindowsGallaryApp.Samples"
+    xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:muxc="using:Microsoft.UI.Xaml.Controls"
+    mc:Ignorable="d"
+    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+
+    <Page.Resources>
+        <CollectionViewSource x:Name="imagesCVS" />
```

Comment:
Is the CVS providing any value here?

Reply:
No it is not providing any value so I can remove it.